sdk/storage/inc/shares/share_file_client.hpp

```diff
@@ -0,0 +1,359 @@
+     * authenticate the source file. If you are copying a file from another storage account, or if
+     * you are copying a file from the same storage account or another storage account, then you
+     * must authenticate the source file or file using a shared access signature. If the source is a
+     * public file, no authentication is required to perform the copy operation. A file in a share
+     * snapshot can also be specified as a copy source.
+     * @param options Optional parameters to copy the content of this file.
+     * @return Azure::Core::Response<StartCopyFileResult> containing the copy related information.
+     */
+    Azure::Core::Response<StartCopyFileResult> StartCopy(
+        std::string copySource,
+        const StartCopyFileOptions& options = StartCopyFileOptions()) const;
+
+    /**
+     * @brief Aborts copying the file specified with the copy ID.
+     * @param copyId The copy identifier provided in the StartCopyFileResult of the original
+     * StartCopy operation.
+     * @param options Optional parameters to abort copying the content of this file.
+     * @return Azure::Core::Response<AbortCopyFileResult> containing the abort copy related
+     * information, current empty but preserved for future usage.
+     */
+    Azure::Core::Response<AbortCopyFileResult> AbortCopy(
+        std::string copyId,
+        const AbortCopyFileOptions& options = AbortCopyFileOptions()) const;
+
+    /**
+     * @brief Gets the properties of a file.
+     * @param options Optional parameters to get the properties of this file.
+     * @return Azure::Core::Response<GetFilePropertiesResult> containing the file properties.
+     */
+    Azure::Core::Response<GetFilePropertiesResult> GetProperties(
+        const GetFilePropertiesOptions& options = GetFilePropertiesOptions()) const;
+
+    /**
+     * @brief Sets the properties of the file.
+     * @param smbProperties The SMB properties to be set to the file.
+     * @param options Optional parameters to set this file's properties.
+     * @return Azure::Core::Response<SetFilePropertiesResult> containing the properties of the
+     * file returned from the server.
+     */
+    Azure::Core::Response<SetFilePropertiesResult> SetProperties(
+        FileShareSmbProperties smbProperties,
+        const SetFilePropertiesOptions& options = SetFilePropertiesOptions()) const;
+
+    /**
+     * @brief Sets the properties of the file.
+     * @param newSizeInByte Byte value of the size the file to resize to. If the specified byte
+     * value is less than the current size of the file, then all ranges above the specified byte
+     * value are cleared.
+     * @param options Optional parameters to resize the file.
+     * @return Azure::Core::Response<ResizeFileResult> containing the properties of the
+     * file returned from the server.
+     */
+    Azure::Core::Response<ResizeFileResult> Resize(
+        int64_t newSizeInByte,
```

Comment:
Do we need `InByte` here? Other length variables dont end up with `InBytes`

Reply:
Removed.